### PR TITLE
README.md - add a little section on the includes to remove a question/roadblock for new users

### PR DESCRIPTION
**Pull request by aaronlippold**
* addes the needed 'vendor' directions for the using the resources.

Signed-off-by: Aaron Lippold <lippold@gmail.com>

### CODE REVIEW

**Comment by clintoncwolfe** [comment]
Hi Aaron, 
  Thanks for the suggestion!  However, vendoring is an optional step, and is not specific to inspec-aws.  You can use the resource pack in several different ways.  Vendoring is a great way to capture and control a set of profile versions at deployment time, especially for production environments, but as this resource pack is in incubator phase, I'm not sure adding vendoring instructions here is appropriate.

@adamleff, thoughts?

**Comment by aaronlippold** [comment]
I was fairly sure you needed to vendor the profile if you did a depends in the inspec.yml but that may be an old requirement that is no longer needed in the current inspec code.

**Comment by adamleff** [comment]
It's confusing because you don't need to vendor in order to generate the required inspec.lock file, which will happen automatically. However, we tell people to run `inspec vendor --overwrite` to update their lock file, when that's not 100% necessary; you could just remove the lock file and then it will get generated again based on your local profile cache.

My recommendation would be to not detail the steps necessary to vendor in this project's README (especially since it should be rather short-lived) but instead just link to our existing documentation on how to use profile dependencies:

https://www.inspec.io/docs/reference/profiles/#profile-dependencies

File: README.md [hunk 1]
```diff
@@ -63,6 +63,13 @@ control "aws-1" do
 end
 ```
 
+### Vendor your Inherited Profile
+
+Your `depends:` above makes your `my-profile` an inherited InSpec profile. (see:[Profile Dependencies](https://www.inspec.io/docs/reference/profiles/index.html#profile-dependencies) in the InSpec docs) Prior to using your `my-profile` make sure you run:
```
Comment (adamleff) [inline README.md]: This is not correct. `my-profile` is the including profile, `aws` is the inherited profile. But again, I think this is out-of-scope for this README and a link to the existing docs is all that's necessary. This is a short-term need anyways.

**Pull request closed without merging**